app/renderer/components/Inspector/SelectedElement.js

```diff
@@ -8,6 +8,8 @@ import { clipboard } from 'electron';
 
 const ButtonGroup = Button.Group;
 
+const tableColumnsStyle = { wordWrap: 'break-all', wordBreak: 'break-all' };
```

Comment:
cannot we define the style in css file?

Reply:
Could 🆗 
Maybe something was wrong when I tried once...